### DISCUSSION TITLE
Using log in credentials in multiple tests

### DISCUSSION THREAD

**Discussion by Alex-passiv** [cypress-io/cypress#14542] [Questions and Help]
Hi, 

Im trying to find a way to have one fixture with my login credentials and then import it to each test. I've tried using fixtures but it keeps saying that the type is undefined (though its coming from a json so it's a string?). I know someone has had to do this before. 

Main use is that I can just set the constants in the tests and then never have to change them and can then test different accounts via one file. Right now I have to change them in each file. Tedious. 

Any ideas?

**Comment by bahmutov** [comment]
Please provide code you are writing, otherwise there is no way anyone knows what is going wrong.

Also, for fixtures please read https://on.cypress.io/fixture documentation. It might be that you are using `this.name` to access a fixture object from an arrow function.
